code/ui/manager/src/components/layout/Layout.types.ts

```diff
@@ -0,0 +1,30 @@
+import type { API_PanelPositions, ViewMode } from '@storybook/types';
+import type React from 'react';
+import type { usePersistence } from './Layout.persistence';
+
+export interface ExposedLayoutState {
+  panelPosition: API_PanelPositions;
+  panel: boolean;
+  sidebar: boolean;
```

Comment:
Personal preference: I like to follow a convention of booleans starting with `is` or `has` or `show` or something like that, rather than just a noun.  I had to dig and find these types to convince myself they were indeed boolean.  I think it would be more clear if these were `showPanel` and `showSidebar`, or similar.

Reply:
`showPanel` suggests a function to me. I prefer `is` or `has` for booleans.